spec/registries/e2e_spec.sh

```diff
@@ -17,7 +17,12 @@ Describe 'Supported Registry'
         OCI_REGISTRY_NAME="$1"
         OCI_REGISTRY_URL="$2"
         OCI_REGISTRY_USERNAME="$3"
-        OCI_REGISTRY_PASSWORD="$4"
+        if [ "$OCI_REGISTRY_USERNAME" = "AWS" ]; then
```

Comment:
Doing this here so we don't print to the GitHub action console (the values in secrets are obfuscated but if we use environment variables or the `shellspec` `When` clause, it will get printed